bluebrain/deployment/gitlab-ci.yml

```diff
@@ -225,7 +225,7 @@ github_information:
     # Activate if checksums fail
     # - spack clean -a
     # Concretize and pre-fetch, afterwards build the environment
-    - spack concretize -f|tee "concretization_${DEPLOY_ENV}.txt"
+    - spack --show-cores=full concretize -f|tee "concretization_${DEPLOY_ENV}.txt"
```

Comment:
Everything looks good to me, but I think @matz-e should look at this.

Reply:
That was me :)

"This is fine"

Reply:
Alright, merging. Thank you both!